appveyor.yml

```diff
@@ -1,18 +1,67 @@
-image:
-  - Visual Studio 2013
-  - Visual Studio 2015
-
-environment:
+environment:  
+  GITHUB_TOKEN:
+    secure: 5JPgkWrjzetioB0LX9ZmMUV+5gk2/4vtkhgvaMVmkoOCNwxaraKuHGYjr6CkD8hR
   MSBUILD_FLAGS: /verbosity:minimal /maxcpucount /nologo
+  matrix:
+  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+    GENERATOR: "Visual Studio 12 2013 Win64"
+    VS_VERSION: vs2013
+  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+    GENERATOR: "Visual Studio 14 2015 Win64"
+    VS_VERSION: vs2015
+
+platform: x64
 
 matrix:
   fast_finish: true
 
-build_script:
- - mkdir build
- - cd build
- - cmake -DRINGMESH_WITH_GRAPHICS:BOOL=ON -DRINGMESH_WITH_UTILITIES:BOOL=ON -DRINGMESH_WITH_TESTS:BOOL=ON -DRINGMESH_WITH_TUTORIALS:BOOL=ON ..
- - cmake --build . --config Debug -- %MSBUILD_FLAGS%
+for:
+-
+  branches:
+    only:
+      - master

```

Comment:
is build_script/test_script/deploy_script run only for master branch ?

Reply:
Or "for:" stands for the "-" lines 19 and 57 ?

Reply:
You have the common configuration before the "for:" and after you have the remaining of the two configurations. The one only for the master (from line 19) and the one for everything else (from line 57).

Reply:
Ok thank you for explaining me this syntax. Great job